Support to the Lua 5.3.2

chentz78 opened the pull request:
…994-2015 Lua.org, PUC-Rio"

chentz78 committed:
Added support to the Lua 5.3; Test result "Lua 5.3.2  Copyright (C) 1994-2015 Lua.org, PUC-Rio"

jdesgats commented on telescope.lua:
```diff
@@ -165,7 +165,7 @@ local function make_assertion(name, message, func)
       for i = 1, nargs do a[i] = tostring(args[i]) end
       for i = nargs+1, num_vars do a[i] = 'nil' end
-      return (assertion_message_prefix .. message):format(unpack(a))
+      return (assertion_message_prefix .. message):format(table.unpack(a))
```
This breaks compatibility with Lua 5.1/LuaJIT (depending on compile time flags). The usual way to handle this is to add:

``` lua
local unpack = unpack or table.unpack
```

At the top of the module.

chentz78 replied:
I did know about the Lua 5.1 compatibility issue. I'll change it.
Thanks!

chentz78 committed:
Fixed the compatibility with Lua 5.1 and Luajit.
Test results:
Lua 5.2.4  Copyright (C) 1994-2015 Lua.org, PUC-Rio
13 tests 13 passed 32 assertions 0 failed 0 errors 0 unassertive 0 pending

Lua 5.3.2  Copyright (C) 1994-2015 Lua.org, PUC-Rio
13 tests 13 passed 32 assertions 0 failed 0 errors 0 unassertive 0 pending

jdesgats commented:
Merged, thanks :)
